webapp/channels/src/sass/components/_modal.scss

```diff
@@ -471,20 +471,6 @@
                 max-height: 100%;
             }
 
-            .modal-close {
```

Comment:
There's a couple CSS changes here because I found some CSS referencing some of these old images, but it doesn't look like there are any elements that use these classes in the app